agr_java_core/src/main/java/org/alliancegenome/api/service/AlleleColumnFieldMapping.java

```diff
@@ -31,7 +31,9 @@ public AlleleColumnFieldMapping() {
         mapColumnAttribute.put(GENE_ALLELE_CATEGORY, entity -> Set.of(entity.getCategory()));
         mapColumnAttribute.put(GENE_ALLELE_VARIANT_TYPE, entity -> {
             if (entity.getVariants() != null) {
-                return entity.getVariants().stream().map(variant -> variant.getVariantType()!=null?variant.getVariantType().getName():"").collect(toSet());
```

Comment:
@jt15 I guess we both fixed this.

Reply:
yes, how long do the updates get reflects on the build site?